pkg/bkctl/bookie/bookie.go

```diff
@@ -0,0 +1,47 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package bookie
+
+import (
+	"github.com/streamnative/pulsarctl/pkg/cmdutils"
+
+	"github.com/spf13/cobra"
+)
+
+func Command(flagGrouping *cmdutils.FlagGrouping) *cobra.Command {
+	resourceCmd := cmdutils.NewResourceCmd(
+		"bookie",
+		"Operations about bookie",
+		"",
+		"bookies")
+
+	commands := []func(*cmdutils.VerbCmd){
+		listCmd,
```

Comment:
let's separate `list` command from all the other commands. because `list` command can be issued to any bookies, however all the other commands are only applied to specific bookies.

Use "bookies" for the operations that are applied to the whole bookkeeper cluster, use "bookie" for the operations that are applied to one specific bookie. For all the operations that are applied to one specific bookie, the bookie id is required because the command should be sent to that bookie directly.

Reply:
Can I do this with another separate pull request?

Reply:
Currently, the bookie id can not be specified using a flag. I will do this with another pull request.